apps/foundation-demo/src/app/misc/header/components/sidebar/sidebar.component.html

```diff
@@ -0,0 +1,7 @@
+<div class="sidebar" [ngClass]="showSidebar ? 'sidebar-show' : 'sidebar-hide'">
+  <span class="sidebar__close" (click)="toggleShowSidebar()"><i class="fa fa-times"></i></span>
+  <div sidebar-item>women's</div>
```

Comment:
Even though it's static at the moment, can we make this a loop just to make it clear that we intend for this to be dynamic?

On second thought... I think the content should be transcluded here.

Reply:
done